CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ option(BUILD_SHARED_LIBS "Build shared avif library" ON)
 
 option(AVIF_ENABLE_WERROR "Treat all compiler warnings as errors" ON)
 
+option(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R "Enable experimental YCgCo-R matrix code" OFF)
```

Comment:
We can consider a single option that enables all experimental features. See https://github.com/AOMediaCodec/libavif/issues/766.